src/tiledb/cloud/client.py

```diff
@@ -148,9 +148,9 @@ def default_user() -> models.User:
     return config.user
 
 
-def default_charged_namespace() -> str:
+def default_charged_namespace(require_action = None) -> str:
```

Comment:
Please add documentation and a type annotation for what `require_action` is. Also, would `required_action` (or just `action`) be a better name?